cards/cardwithShadow.css

```diff
@@ -0,0 +1,105 @@
+*{
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
+    font-family: sans-serif;
+    letter-spacing: 1px;
+}
+:root{
+        --card-background-color:aliceblue;
+        --card-border-color:black;
+        --card-badge-bg-col:#BFC205;
+        --card-badge-rating-bg:green;
+        --card-primary-color:#F6830F;
+        --card-box-shadow-col:grey;
+        --card-btn-bg:#082032;
+    }
+.card-legend{
+    width:20%;
+    border:1px solid var(--card-border-color);
+    background-color: var(--card-background-color);
+    position: relative;
+    border-radius:10px;
+    overflow:hidden;
+
+}
+.card-legend-shadow{
+    box-shadow: 10px 10px 5px var(--card-box-shadow-col);
+}
+.para-card .heading-card{
+    padding-top:0.2rem;
+    padding-bottom:0.2rem;
+    font-weight: 700;
+    margin-left:0.3rem;
+}
+.para-card .subheading-card{
+    padding-top:0.2rem;
+    padding-bottom:0.2rem;
+    font-weight: 200;
+    margin-left:0.3rem;
+}
+.para-card .price-card{
+    position: absolute;
+    right:8px;
+    bottom:45px;
+    font-weight: 200;
+}
+.para-card .rating-card .rating-h{
+    background-color: var(--card-badge-rating-bg);
+    display: inline;
+    padding:0.3rem;
+    padding-left:0.5rem;
+    padding-right:0.5rem;
+    border-radius:5px;
+    color:var(--card-background-color);
+    position: absolute;
+    right:8px;
+    bottom:70px;
+
+}
+.badge-card{
+    width:35%;
+    height: auto;
+    background-color: var(--card-badge-bg-col);
+    padding:0.7rem;
+    padding-left:1rem;
+    padding-right: 1rem;
+    text-align: center;
+    border-radius:5px;
+    position: absolute;
+    top: 1rem;
+
+}
+.image img{
+    width: 100%;
+    height: 30%;
+}
+.para-card p{
+    text-align: justify;
+    padding: 0.3rem;
+    margin-top:0.3rem;
+    margin-bottom:0.3rem;
+}
+.card-btn{
+    padding-top:0.2rem;
+    background-color: var(--card-btn-bg);
+    font-weight: bold;
+    font-size:1.5rem;
+    color:var(--card-background-color);
+    cursor:pointer;
+    text-align: center;
+    border-radius: 10px;
+    padding-right: 1rem;
+    padding-left:1rem;
+    margin-bottom:0.5rem;
+    padding-bottom:0.3rem;
+
+}
+.card-btn:hover{
+    background-color: var(--card-primary-color);
+    margin-top:0.4rem;
+}
+
+.card-button{
+    text-align: center;
```

Comment:
You can provide height to "card-button" <div> as its overlapping with price <div>